TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift

```diff
@@ -216,6 +217,24 @@ class TradeItPortfolioAccountDetailsViewController: TradeItViewController, Trade
         let order = self.provideOrder(forPortfolioPosition: portfolioPosition, account: portfolioPosition?.linkedBrokerAccount, orderAction: orderAction)
         self.tradingUIFlow.presentTradingFlow(fromViewController: self, withOrder: order)
     }
+    
+    func proxyVoteWasTapped(forPortfolioPosition portfolioPosition: TradeItPortfolioPosition?) {
+        portfolioPosition?.getProxyVoteUrl(
```

Comment:
We may want to add a spinner here, the time we call trade it backend to get the url?